src/index.ts

```diff
@@ -44,6 +44,8 @@ export interface CookieOptions {
   signed?: boolean
 }
 
+export type Body = mixed | mixed[] | Buffer
```

Comment:
Defining a `Body` type is not necessary, `t.mixed` is a top type, i.e. _every_ type is a subtype of `mixed`, included `Buffer`